Jenkinsfile

```diff
@@ -493,7 +492,56 @@ pipeline {
             }
         }
 
+        // the Django Unit Tests stage runs backend unit tests using a test DB that is
+        // created and destroyed afterwards. 
+        stage('DEV - Django Unit Tests') {
+            when {
+                expression { env.CHANGE_TARGET != 'master' && env.CHANGE_TARGET != 'demo' }
```

Comment:
We need to make sure the Django tests are run in all pipelines.  Let's remove the `when { expression ... }` condition blocks from all the "DEV" stages so every PR gets a dev environment with the testing.

Reply:
so keep `expression { env.CHANGE_TARGET != 'master' }` ? 
@stephenhillier

Reply:
I think we need to remove `env.CHANGE_TARGET != 'master'` as well (the whole `when` block).  The unit tests need to run every time the image is rebuilt, so there shouldn't be any `when { }` conditions here

Reply:
in the "olden" days, master branch was only for TEST/PROD deploy, while everything else only deployed to DEV.   I haven't kept up with the pipeline changes so I may be mistaken.. but don't master changes only proceed once DEV/TEST pipelines have passed?  Are we running the complete unit tests (and autocreation of test_gwells DB) even in master?

Reply:
`env.CHANGE_TARGET` is used throughout (e.g. https://github.com/bcgov/gwells/blob/3a55ada221c095e1dac30dd8903d0af1501398bf/Jenkinsfile#L366) so if we decide to make the Unit Tests run for all `!= master` we need to change them all consistently.  I'm beginning to wonder if I went down the wrong path in moving Unit Tests post-deployment.   Can I set up con-call tomorrow (or even today if @Sybrand  is not too sick)?  This PR is getting too big.

Reply:
done.. 'Django Unit Tests' will run in all DEV pipelines (but != master)